tests/functional/behave_features/common/utils/e2e_templates.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+"""String templates to be used for generating various files throughout E2E testing."""
+
+# The base OWNERS file string template.
+base_owners_file: str = """\
+chart:
+  name: ${chart_name}
+  shortDescription: Test chart for testing chart submission workflows.
+publicPgpKey: ${public_key}
+providerDelivery: ${provider_delivery}
+users:
+- githubUsername: ${bot_name}
+vendor:
+  label: ${vendor}
+  name: ${vendor_pretty}
+"""
```

Comment:
Note For Reviewers:

A version of this base_owners_file template exists in chart_certification.py, and will be superseded by this shared template in the future.